src/components/button/nz-button.component.spec.ts

```diff
@@ -123,13 +123,12 @@ describe('NzButton', () => {
       expect(buttonDebugElement.nativeElement.classList.contains('custom-class')).toBe(true);
     });
 
-    it('should handle a click on the button', () => {
+    it('should handle a click on the button', fakeAsync(() => {
       buttonDebugElement.nativeElement.click();
       expect(testComponent.isLoading).toBe(true);
-      setTimeout(_ => {
```

Comment:
This is failing randomly in my local machine, fixed now. (Likely not work before as sync function)

Reply:
`tick` is fine